pkg/runner/command.go

```diff
@@ -7,7 +7,7 @@ import (
 )
 
 // testCmd configures default Service Account as a kubeconfig, runs openshift-tests, and serves results over HTTP
-const testCmd = `
+const testCmd = `#!/bin/bash
```

Comment:
We should explicitly call this out and ensure documentation states the test container images require `bash` to be installed. 

Often times containers don't actually have `bash` installed, only sh. This isn't so much a critique on the PR but raising the need to document this once addon testing starts.

Reply:
Agreed here -- I'll add this to the doc.